src/coreclr/src/jit/emit.cpp

```diff
@@ -4622,10 +4622,18 @@ unsigned emitter::emitEndCodeGen(Compiler* comp,
     }
 #endif
 
-    if (emitConsDsc.align16)
+    // This restricts the emitConsDsc.alignment to: 1, 2, 4, 8, 16, or 32
```

Comment:
Presumably bytes (would be good to be explicit).

Reply:
Fixed.